grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -78,6 +78,10 @@ class StockMovementController {
         render(view: "/common/react", params: params)
     }
 
+    def createInboundV2() {
```

Comment:
Is this one required? Can we just use the regular `createInbound` action?

Reply:
I agree with you, because this is a feature branch I don't see anything against replacing the old one.

Reply:
if we were to replace it, Sebastian would not be able to access the old workflow to compare those while working. We decided it would be easier for him to do it this way, and this will be removed at the end of the development (probably when the PR from feature branch to develop is created)

Reply:
@kchelstowski That is not what I meant here. I rather wanted to move it to the urlmapping, to handle that part and point to proper (the old one) action, to avoid adding this `V2` in the controller.

Reply:
@awalkowiak ok now I get it, I don't know why I didn't realize we could actually use two URLs and map to the same action, and let it be handled further by the react router.